server/config.json

```diff
@@ -19,8 +19,8 @@
       "limit": "100kb"
     },
     "cors": false,
-    "errorHandler": {
-      "disableStackTrace": false
+    "strongErrorHandler": {
+      "debug": false
```

Comment:
The keys are referring to middleware module/file names. I think you should use `strong-error-handler` here. Also since loopback-workspace is never used in production-like mode, I think `debug` can be always set to `true`.